### PR TITLE
Add "Freedom Ipsum" text generator option

### PR DESCRIPTION
**Pull request by lynnd335**
Add "[Freedom Ipsum](http://ameripsum.win/)" ultra-patriotic filler text to FFaker ... reference MD file updated accordingly.

### CODE REVIEW

File: REFERENCE.md [hunk 1]
```diff
@@ -38,6 +38,10 @@
  * [FFaker::DizzleIpsum](#ffakerdizzleipsum)
  * [FFaker::Education](#ffakereducation)
  * [FFaker::Food](#ffakerfood)
+<<<<<<< HEAD
```
Comment (rocknruby) [inline REFERENCE.md]: please remove your conflicted REFERENCE.md and generate it again

File: ffaker [hunk 2]
```diff
@@ -0,0 +1 @@
+Subproject commit b1f50321facff2145045b561cdb2e56033240643
```
Comment (rocknruby) [inline ffaker]: ?

**Comment by rocknruby** [comment]
Same here, could you please rename/squash your commits to something useful at the end 👍

File: ffaker (hunk 2 above)
Comment (lynnd335) [inline ffaker]: No idea... not sure how to remove it either

**Comment by AlexWayfer** [comment]
There are conflicts. :pensive: 

Can you resolve them please?

**Comment by AlexWayfer** [comment]
> @Volosh1n closed this 37 minutes ago

Why?

**Comment by Volosh1n** [comment]
> > @Volosh1n closed this 37 minutes ago
> 
> Why?

4 years old PR without any activity or responses from author. Is it bad?

**Comment by AlexWayfer** [comment]
> > > @Volosh1n closed this 37 minutes ago
> > 
> > 
> > Why?
> 
> 4 years old PR without any activity or responses from author. Is it bad?

I'm not sure, it's subjective, but as you wish. I don't like closings without a description (reason).

**Comment by Volosh1n** [comment]
> > > > @Volosh1n closed this 37 minutes ago
> > > 
> > > 
> > > Why?
> > 
> > 
> > 4 years old PR without any activity or responses from author. Is it bad?
> 
> I'm not sure, it's subjective, but as you wish. I don't like closings without a description (reason).

Yes, my bad. Let's keep it open, sure we made enough notifications to all PR's  subscribers - maybe work will be continued, who knows.

**Pull request merged**